# HG changeset patch
# User Yury Ivanovich <Yury.Ivanovich@linuxmail.org>
# Date 1759147798 -7200
# Node ID de0f81aa90d9a8c19f86368ce3146d93739607c2
# Parent  7079f6d9c86f7923689043dbbf4fafcbf343b309
Bug 1984814 - Codespell mailnews/. r=mkmelin

Note the bug in ParseMsgHeaders().

Differential Revision: https://phabricator.services.mozilla.com/D262234

diff --git a/mailnews/local/src/nsParseMailbox.cpp b/mailnews/local/src/nsParseMailbox.cpp
--- a/mailnews/local/src/nsParseMailbox.cpp
+++ b/mailnews/local/src/nsParseMailbox.cpp
@@ -118,9 +118,9 @@ RawHdr ParseMsgHeaders(mozilla::Span<con
       net_ParseContentType(hdr.Value(raw), contentType, charset, &hasCharset);
       if (hasCharset) {
         out.charset = charset;
       }
-      if (contentType.LowerCaseEqualsLiteral("multpart/mixed")) {
+      if (contentType.LowerCaseEqualsLiteral("multipart/mixed")) {
         out.flags |= nsMsgMessageFlags::Attachment;
       }
     } else if (n.LowerCaseEqualsLiteral("date")) {
       nsCString dateStr = hdr.Value(raw);
